src/Pos/Delegation/Class.hs

```diff
@@ -56,9 +54,6 @@ data DelegationWrap = DelegationWrap
       -- a good approximation.
     , _dwEpochId           :: EpochIndex
       -- ^ Epoch index 'DelegationWrap' is correct in relation to.
-    , _dwThisEpochPosted   :: HashSet PublicKey
```

Comment:
I don't think a single read from DB can be faster. But it also shouldn't be much slower.
What can be slow is to read all ids in `dlgVerifyBlocks` in tests. Profiling will answer.